### PR TITLE
Remove useless uncheckedCast in PHP tests

### PR DESCRIPTION
**Pull request by bernardnormier**
It's also unclear why we prefer this syntax for uncheckedCast in the tests, vs the more standard call to the static method on the generated Helper.

### CODE REVIEW

**Comment by pepone** [comment]
> vs the more standard call to the static method on the generated Helper.

I think we picked this because it was simpler when we supported both namespace and no namespace builds.

File: php/test/Ice/binding/Client.php [hunk 1]
```diff
@@ -53,7 +52,7 @@ function allTests($helper)
 
         $com->deactivateObjectAdapter($adapter);
 
-        $test3 = $test1->ice_uncheckedCast("::Test::TestIntf");
+        $test3 = $test1;
         test($test3->ice_getConnection() == $test1->ice_getConnection());
         test($test3->ice_getConnection() == $test2->ice_getConnection());
```
Comment (pepone) [inline php/test/Ice/binding/Client.php:57]: Didn't we remove this part of the test in other mappings?

File: php/test/Ice/binding/Client.php [hunk 2]
```diff
@@ -285,7 +284,7 @@ function allTests($helper)
 
         $com->deactivateObjectAdapter($adapter);
 
-        $test3 = $test1->ice_uncheckedCast("::Test::TestIntf");
+        $test3 = $test1;
```
Comment (pepone) [inline php/test/Ice/binding/Client.php:287]: ditto just call `$test1->ice_getConnection()`

**Review by pepone — APPROVED** [review]

**Review by externl — APPROVED** [review]

File: php/test/Ice/binding/Client.php (hunk 1 above)
Comment (bernardnormier) [reply php/test/Ice/binding/Client.php:57]: Fixed.

File: php/test/Ice/binding/Client.php (hunk 2 above)
Comment (bernardnormier) [reply php/test/Ice/binding/Client.php:287]: Fixed

**Pull request merged**